langlib/lang.test/src/main/ballerina/src/lang.test/test_functions.bal

```diff
@@ -18,24 +18,57 @@
 #
 # + expected - expected value
 # + actual - actual value
-public function assertValueEqual(anydata expected, anydata actual) = external;
+# + message - Assertion error message
+public function assertEqual(anydata actual, anydata expected, string? message = ()) = external;
```

Comment:
Should this be `assertEquals` and `assertNotEquals` instead, since that seems to be the convention? Testerina also uses the same.

Shall we also move `assertNotEquals` to immediately after this function?